gradoop-common/src/main/java/org/gradoop/common/model/impl/id/GradoopIdSet.java

```diff
@@ -188,11 +192,12 @@ public boolean containsAll(GradoopIdSet other) {
    * @param other the ids to look for
    * @return true, iff all specified ids are contained in the set
    */
-  public boolean containsAll(Set<GradoopId> other) {
+  @Override
+  public boolean containsAll(Collection<?> other) {
```

Comment:
What I actually meant with the comment to make the argument a `Set<?>` was the following scenario:
```
this: {1,2,3,4}
other: {1, 1, 2, 3, 4}
```
the additional check would result in `false` even if all elements are contained in `this` set

Reply:
Ah, yes, you are right. The `size` check still assumes a set argument. We can't rely on it anymore but we also can't use `Set` as argument type because the `Collection` interface dictates this signature. I will check the argument type before using the `size` heuristics.